c++/src/capnp/compiler/node-translator.c++

```diff
@@ -698,6 +698,10 @@ void NodeTranslator::compileNode(Declaration::Reader decl, schema::Node::Builder
   if (decl.hasDocComment()) {
     di.setDocComment(decl.getDocComment());
   }
+  if (decl.which() != Declaration::ANNOTATION) {
```

Comment:
Why not for annotations?

Reply:
This is just my mistake. I misunderstood, so I removed this unnecessary check.